src/components/Search/SearchBar/SearchBar.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { MemoryRouter } from "react-router-dom";
+import ErrorBoundary from "HOC/ErrorBoundary/ErrorBoundary";
+import SearchBar from "components/Search/SearchBar/SearchBar";
+import { renderWithProviders, screen, fireEvent } from "utils/test-utils";
+import { moviesAction } from "features/movies/moviesSelector";
+import { waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+jest.mock("features/movies/moviesSelector", () => ({
+  moviesAction: {
+    resetState: jest.fn(),
+  },
+}));
+
+describe("SearchBar testing", () => {
+  test("snapshot for SearchBar", () => {
+    const { baseElement } = renderWithProviders(
+      <ErrorBoundary>
+        <MemoryRouter>
+          <SearchBar />
+        </MemoryRouter>
+      </ErrorBoundary>
+    );
+    expect(baseElement).toMatchSnapshot();
+  });
+  test("fire input event, search event and calling fetchMovies", async () => {
```

Comment:
This one doesn't test what it says it does. I'd say the most critical part here is that params are added to the URL, we can test it here.